src/AudioAnalysisTools/Events/Drawing/EventDrawer.cs

```diff
@@ -0,0 +1,78 @@
+// <copyright file="EventDrawer.cs" company="QutEcoacoustics">
+// All code in this file and all associated files are the copyright and property of the QUT Ecoacoustics Research Group (formerly MQUTeR, and formerly QUT Bioacoustics Research Group).
+// </copyright>
+
+namespace AudioAnalysisTools.Events.Drawing
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Text;
+    using SixLabors.Fonts;
+    using SixLabors.ImageSharp;
+    using SixLabors.ImageSharp.Processing;
+    using static Acoustics.Shared.ImageSharp.Drawing;
+
+    public static class EventDrawer
+    {
+
+        /// <summary>
+        /// Draws a "score" indicator on the left edge of an event.
+        /// </summary>
+        /// <param name="graphics">The image context to draw to.</param>
+        /// <param name="options">The event rendering optons to use.</param>
+        /// <param name="@event">The event for which to draw the score indicator.</param>
+        public static void DrawScoreIndicator(this SpectralEvent @event, IImageProcessingContext graphics, EventRenderingOptions options)
+        {
+            if (!options.DrawScore)
+            {
+                return;
+            }
+
+            // TODO: add a Interval<double> ScoreRange property to EventCommon
+            // so we can properly normalize this value to the unit value.
+            // For now, we just assume it is normalized to [0,1].
+            //var clampedScore = @event.Score.Clamp(0, 1);
+            var normalisedScore = @event.ScoreNormalised;
+
```

Comment:
Remove comment - this has been no?

Also, the value should be clamped unless ScoreNormalised guarantees a value of [0,1]

Reply:
This issue has been fixed in latest commit.
Comment has been changed and clamp shifted to the Draw method.